server/models/index.js

```diff
@@ -1,37 +1,3 @@
-
-const fs        = require('fs');
-const path      = require('path');
-const Sequelize = require('sequelize');
-const basename  = path.basename(module.filename);
-const env       = process.env.NODE_ENV || 'development';
-const config    = require(`${__dirname}/../config/config.json`)[env];
```

Comment:
Why do you have to move all these from your `models/index.js` directory and where do you move it to?

Reply:
I moved it because eslint informed me that they were assigned but not used so i moved them to each model files that required them